hawc_hal/HAL.py

```diff
@@ -47,16 +47,19 @@ class HAL(PluginPrototype):
     :param flat_sky_pixels_size: size of the pixel for the flat sky projection (Hammer Aitoff)
     """
 
-    def __init__(self, name, maptree, response_file, roi, flat_sky_pixels_size=0.17):
+    def __init__(self, name, maptree, response_file, roi, flat_sky_pixels_size=0.17, n_transits=None):
 
         # Store ROI
         self._roi = roi
 
+        # optionally specify n_transits
+        assert (n_transits==None or n_transits > 0.0), "You must specify n_transits >0"
```

Comment:
Hm. This error message is a bit misleading, as `None` is also a valid value for `n_transits`. Also, do we really need to be that harsh? I think it would be fine to interpret n_transits <= 0 to mean that the user wants to use the transits from the map, similar to what many apps in aerie are doing.

Reply:
I agree that using negative numbers might be a bit smoother logically. I can change that.

Reply:
This has been fixed in the recent commits